tests/openssh.rs

```diff
@@ -17,6 +17,21 @@ async fn it_connects() {
     session.close().await.unwrap();
 }
 
+#[tokio::test]
+#[cfg_attr(not(ci), ignore)]
+async fn control_dir() {
+    let dirname = std::path::Path::new("control-test");
+    if !dirname.exists() {
```

Comment:
I think we can probably `assert!` that this directory does not already exist. If it does, the test wouldn't actually test anything useful.